packages/app/src-tauri/src/commands/ftp.rs

```diff
@@ -120,24 +129,22 @@ async fn ftp_upload_progress_task(
 
     // Trasfer phase: 80-88%
 
-    let min_chunck_size = SIUnit::new(6., SISize::Megabyte);
-    let mut file_stream = FileStream::new(local_path)
-        .await?
-        .set_mode(get_chunk::ChunkSize::Bytes(min_chunck_size.into()));
+    let mut file_stream = FileStream::new(local_path).await?;
 
     let mut writer = ftp_stream.put_with_stream(filename).await?.compat_write();
 
-    let chunk_number = <SIUnit as Into<f64>>::into(
-        SIUnit::auto(file_stream.get_file_size()) / min_chunck_size.into(),
-    )
-    .ceil() as u16;
-    let delta_progress = 80 / (chunk_number) as u8;
+    let mut temp_progress: i32 = 0;
+    let temp_progress_max: i32 = file_stream.get_file_size() as i32 / 80;
 
     while let Ok(Some(chunk)) = file_stream.try_next().await {
         select! {
             res = writer.write_all(&chunk) => {
+                temp_progress += 1;
```

Comment:
@newfla any suggestion for improving this part?

Reply:
The reasoning isn't very clear to me :/ 
 I would do it like this:
`let incresead_file_size = file_stream.get_file_size()*100./80.;`
`let delta_progess = chunk.len() * 100 / incresead_file_size as usize;`